plugins/org.jkiss.dbeaver.ext.firebird/src/org/jkiss/dbeaver/ext/firebird/model/FireBirdFieldType.java

```diff
@@ -65,14 +70,14 @@ public String getName() {
     }
 
     public static FireBirdFieldType getById(int id, int subTypeId) {
-        if (id == 16) {
+        if (id == 16 || id == 8) {
```

Comment:
But `INTEGER` doesn't share its id with any other type. Is this documented somewhere?